apps/mdn/mdn-aws/k8s/kuma.base.yaml.j2

```diff
@@ -87,6 +87,8 @@
                   key: elasticsearch-url
             - name: FOUNDATION_CALLOUT
               value: "{{ KUMA_FOUNDATION_CALLOUT }}"
+            - name: INTERACTIVE_EXAMPLES_BASE
+              value: "{{ KUMA_INTERACTIVE_EXAMPLES_BASE }}"
```

Comment:
This needs to be in the environment for the other regions (``portland/prod.sh``, ``portland/stage.sh``, etc.). This can be done either be adding a line to those files:

```
export KUMA_INTERACTIVE_EXAMPLES_BASE=https://interactive-examples.mdn.mozilla.net
```

or by adding a default in the ``Makefile``.

Reply:
Oh my, good catch! How myopic of me! Thanks @jwhitlock!